code/game/objects/items/ego_weapons/non_abnormality/shi.dm

```diff
@@ -118,3 +118,26 @@
 							TEMPERANCE_ATTRIBUTE = 100,
 							JUSTICE_ATTRIBUTE = 80
 							)
+
+//Specialist Shi Blades (I had the sprites.)
+/obj/item/ego_weapon/city/shi_assassin/sakura
+	name = "shi association sakura blade"
+	desc = "A unique specialized assassin blade that is used by Shi Section 2. Created for highly armored targets, this one deals white damage"
+	icon_state = "shi_sakura"
+	damtype = WHITE_DAMAGE
+	armortype = WHITE_DAMAGE
+
+/obj/item/ego_weapon/city/shi_assassin/serpent
+	name = "shi association seperant blade"
+	desc = "A unique specialized assassin blade that is used by Shi Section 2. Created for highly armored targets, this one deals black damage"
+	icon_state = "shi_serpent"
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+
+/obj/item/ego_weapon/city/shi_assassin/yokai
```

Comment:
This one already does pale, it's special should be buffed accordingly to make up for the 24 lost force, 48 when the ability is used.

Reply:
Fixed